### DISCUSSION TITLE
How to convert "extxyz" format into deepmd data

### DISCUSSION THREAD

**Discussion by oskfuta** [deepmodeling/dpdata#577] [Q&A]
I already have my own datasets whose format is not supported by dpdata, then I used my script to convert those datasets into `extxyz` format. However, there was an issue when I tried to use dpdata code to convert `extxyz` format. My script is shown below:

```
import dpdata 
import numpy as np

data = dpdata.MultiSystems.from_dir(dir_name="./files/", file_name="*.xyz", fmt="quip/gap/xyz")
```
Its output:

> TypeError: 'str' object is not a mapping

Do you have any idea to transform the extxyz format?

**Comment by composites123** [comment]
You can upload part of your data in the format you speak.
